src/test/java/SampleTest.java

```diff
@@ -1,13 +1,18 @@
+import Base.BaseTest;
 import pages.MainPage;
 import org.junit.jupiter.api.Test;
 
-public class SampleTest extends BaseTest{
-
-    private MainPage mainPage;
+public class SampleTest extends BaseTest {
 
     @Test
     public void Test() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.